packages/playground/website-deployment/apply-blueprints-mirror-update.sh

```
@@ -0,0 +1,40 @@
+
+echo Syncing Blueprints mirror to production
+mkdir -p "$MIRROR_DIR"
+rsync -av --delete --no-perms --omit-dir-times ~/updated-blueprints-mirror/ "$MIRROR_DIR/"
+
+# A copy served directly by Nginx would shadow the PHP-served one without CORS headers.
+rm -rf /srv/htdocs/blueprints
+
+echo Purging edge cache
+curl -sS -X POST -H "Auth: $ATOMIC_SITE_API_KEY" "$SITE_API_BASE/edge-cache/$ATOMIC_SITE_ID/purge" \
+        > /dev/null \
+        && echo "Edge cache purged" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)